tests/config.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
For extra credit, perhaps you could file a short issue with them noting the problem